### ISSUE TITLE
Possibly need to patch `fetch` out of `@figma/plugin-typings` like you did with `console`

### ISSUE THREAD

**Issue by romannurik** [yuanqing/create-figma-plugin#98]
Recently I've been getting this error, showing a conflict between TypeScript's built-in DOM lib and `@figma/plugin-typings`:

```
    node_modules/@figma/plugin-typings/index.d.ts:28:9 - error TS2451: Cannot redeclare block-scoped variable 'fetch'.

    28   const fetch: (url: string, init?: FetchOptions) => Promise<FetchResponse>
               ~~~~~

      node_modules/typescript/lib/lib.dom.d.ts:17577:18
        17577 declare function fetch(input: RequestInfo, init?: RequestInit): Promise<Response>;
                               ~~~~~
        'fetch' was also declared here.
```

I noticed as part of the build you strip out `console` from `@figma/plugin-typings` because of the conflict w/ DOM; may need to do the same for `fetch`.

**Comment by romannurik** [comment]
Worth noting, manually commenting out the `const fetch: ...` line out of `node_modules/@figma/plugin-typings/index.d.ts` results in a successful build

**Comment by yuanqing** [comment]
Thanks for flagging this! It looks like the `fetch` type declaration was added to `@figma/plugin-typings` wef `1.44.0`. In the meantime, the workaround is to pin to any version before that (eg. `1.43.0`)

**Comment by yuanqing** [comment]
The fix for this issue will be included in the next release

**Comment by yuanqing** [comment]
Fixed in `1.9.1`

**Comment by romannurik** [comment]
Thank you!
